# add_two_numbers_wrong.py

"""
Demonstrates a simple bug that can arise from trying to add two "numbers"
together which are in face strings
"""

# ask the user for the numbers
first_number = input("pick a number, any number ")
second_number = input("now pick another one ")

# add the two numbers
result = first_number + second_number

# print the result
print(result)
